### DISCUSSION TITLE
Trying to emulate running windows 10 on ARM on intel x86_64 fails.

### DISCUSSION THREAD

**Discussion by fenginsc** [utmapp/UTM#3544] [UTM Help (macOS)]
**Describe the issue**
Attempts to emulate running Windows 10 ARM64 on Intel x86_64 failed.
The UEFI boot interface is stuck and cannot enter the installer. I downloaded the ISO from uupdump.net.

**Configuration**
* UTM Version:  3.0.4
* OS Version:  11.6
* Intel or Apple Silicon?  intel

**Comment by conath** [comment]
Did you "press any key" to boot from the ISO?

**Reply by fenginsc** [reply]
Yes, when the interface says "press any key to boot from cd or dvd", I press enter, then the utm logo interface is displayed, but I waited for hours and nothing happened.

**Reply by conath** [reply]
I have seen this problem with Windows ARM without the `,virtualization=on` added to additional machine properties. However as far as I know this is no longer required in UTM 3 - it leads to an error message if used (I am on macOS 12.1, though).

**Reply by fenginsc** [reply]
I upgraded to 12.2 monterey latest version and it still doesn't work, waited 30 minutes and nothing happened, or do I need to wait longer?

**Reply by conath** [reply]
Can you try adding the `,virtualization=on` text to the Additional Machine Properties in System => Show Advanced?

**Reply by fenginsc** [reply]
I tried adding that property and now it works, thank you very much, my version is UTM 3.0.4, I think the next version might need to make this property default to make this virtual machine work correctly, so that no more Added it manually, thank you again.

**Reply by conath** [reply]
@osy Do you think we should add this parameter `,virtualization=on` to ARM VMs running on x86 host?
